src/languages/swift.js

```diff
@@ -67,6 +67,9 @@ export default function(hljs) {
     variants: [
       {begin: /"""/, end: /"""/},
       {begin: /"/, end: /"/},
+      {begin: /#"/, end: /"#/},
+      {begin: /##"/, end: /"##/},
+      {begin: /###"/, end: /"###/},
```

Comment:
Can such strings contain unescaped newlines?

Reply:
Same subst and backlash escape rules apply to raw strings as regular strings or is that different?

Reply:
No, but you can have raw multiline strings (wrapped in `#"""` and `"""#`). I also tried those and they highlight correct for me, with the rules above. Same goes for strings with interpolation or escaped characters.

Reply:
I'm confused what makes them "raw"... if this has no bearing on the escaping, etc?

Reply:
FYI, here are all the code examples in my document that use raw strings:

```swift
print(#"Use \t to insert a tab."#)

print(#"""
Use \t to insert a tab. The following example:

print("\tHello")

prints:

    Hello
"""#)

print(##"Use #" and "# to delimit a raw string."##)

print(#"""
Use \t to insert a tab. The following example:

print("\tHello")

prints:

\#tHello
"""#)

print(#"Hi \#(player), your score is: \#(score)."#)
```

These all highlight correctly with the new rules.

Reply:
What makes them raw is that they interpret "special characters" (such as `\t`) as normal character sequences (backslash and t) instead of their special meaning (tab). However, you can still use escape sequences and string interpolation by adding the same number of hashes after the backslash.

Reply:
I think we need more markup tests... sounds like there is more to be done here... at the very least we don't match normal escapes (as with every other string)... so we should have a negative test case for that... (showing that it workes properly, NOT highlighting what would be an escape)

And we should have some test cases of this new variant of escaping (and then decide whether to handle it or not)... if not then a comment in the test would explain, etc.